### PR TITLE
fix dates for coffee chat 

### PR DESCRIPTION
**Pull request by commit111**
Removed actual dates (such as October 27th) and expired Luma invite links for Defang Coffee Chat/Townhall in `knowledge.json` as an attempt to avoid it giving out outdated dates/times for these events.

May still need some fixing elsewhere in order to for this change to take effect. Perhaps @Chrisyhjiang might know?

### CODE REVIEW

**Comment by commit111** [comment]
Realized the RAG chatbot updates the knowledge base automatically when docs change.

**Comment by commit111** [comment]
Okay, another approach to take is to add the current date to the pre-prompt.

**Comment by commit111** [comment]
Okay, updated the pre-prompt. I also reverted the knowledge base changes to original since they get overridden anyway.

File: app/rag_system.py [hunk 1]
```diff
@@ -70,6 +71,7 @@ def answer_query_stream(self, query):
                     "When the user says 'you', 'your', or any pronoun, interpret it as referring to Defang with context of Defang. "
                     "If the user's question involves comparisons with or references to other services, you may use external knowledge. "
                     "However, if the question is strictly about Defang, you must ignore all external knowledge and only utilize the given context. "
+                    "Today's date is " + date.today().strftime('%B %d, %Y') + "."
```
Comment (lionello) [inline app/rag_system.py:74]:
```suggestion
                    "Today's date is " + date.today().strftime('%B %d, %Y') + ". "
```

**Review by lionello — APPROVED** [review]

**Pull request merged**